frontend/packages/data-portal/app/hooks/useFilter.ts

```diff
@@ -125,46 +125,57 @@ export function useFilter() {
       ...getFilterState(searchParams),
 
       reset() {
-        setSearchParams((prev) => {
-          Object.values(QueryParams).forEach((param) => prev.delete(param))
-          prev.delete('page')
-
-          return prev
-        })
+        setSearchParams(
+          (prev) => {
+            Object.values(QueryParams).forEach((param) => prev.delete(param))
+            prev.delete('page')
```

Comment:
wouldn't this already be covered since `QueryParams.Page` would be deleted by the above line?

Reply:
probably didn't realize it was included in `QueryParams`